### PR TITLE
Templates: Fix typo in Travel template

### LINKED ISSUE #6601: Typo on Travel Section page layout
## Bug Description

There is a typo in the default text of the Travel Section page layout. The text in the lower half of the layout currently reads: "Tips and tricks for caturing [sic] this iconic city in a day." It should read "Tips and tricks for capturing this iconic city in a day."

## Steps to Reproduce

1. In the Story Editor, choose the Page Layouts tab
2. Select the Travel Section layout

## Screenshots

image

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by BrittanyIRL**
## Context

There was a typo!

## Summary

Change spelling of `caturing` to `capturing` on page 5 of travel template.

## User-facing changes

Spells `capturing` correctly. 
Screen Shot 2021-03-04 at 11 21 10 AM

## Testing Instructions

See that page 5 of travel template has updated text on bottom to read "capturing", see that same change is reflected for page layouts (it's a "section" type page to make finding it easier)

### QA

- [ ] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1.

### UAT

- [ ] UAT should use the same steps as above.

This PR can be tested by following these steps:

1.

### Does this PR have a security-related impact?

No

### Does this PR change what data or activity we track or use?

No

### Does this PR have a legal-related impact?

No

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testiing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [ ] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [ ] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #6601

### CODE REVIEW

File: packages/templates/src/raw/travel.json [hunk 1]
```diff
@@ -1595,7 +1629,7 @@
           "focalX": 50,
           "focalY": 50,
           "id": "7dae1bd9-9f88-4ae8-93eb-f3d908cace09",
-          "content": "<span style=\"color: rgba(255, 255, 255, 1)\">Tips and tricks for caturing this iconic city in a day.</span>",
+          "content": "<span style=\"color: #fff\">Tips and tricks for capturing this iconic city in a day.</span>",
```
Comment (samwhale) [inline packages/templates/src/raw/travel.json:1632]: lol

**Review by samwhale — APPROVED** [review]

**Review by embarks — APPROVED** [review]

File: packages/templates/src/raw/travel.json [hunk 2]
```diff
@@ -1,4 +1,12 @@
 {
+  "current": "22081c00-a02f-4085-84de-0f4a31253222",
+  "selection": ["7dae1bd9-9f88-4ae8-93eb-f3d908cace09"],
```
Comment (swissspidy) [inline packages/templates/src/raw/travel.json]: 🤔 

Wouldn't that cause unexpected selection when a user tries to use a template?

Comment (BrittanyIRL) [reply packages/templates/src/raw/travel.json]: my bad, grabbed the updated template while selection was present. fixed.

**Review by maxyinger — APPROVED** [review]

File: packages/templates/src/raw/travel.json [hunk 3]
```diff
@@ -753,7 +768,13 @@
       ],
       "type": "page",
       "id": "3cdf0fde-f515-477d-8688-472c4cc5a8b3",
-      "backgroundColor": { "color": { "r": 255, "g": 255, "b": 255 } },
+      "backgroundColor": {
```
Comment (swissspidy) [inline packages/templates/src/raw/travel.json:771]: Might want to run prettier over this to avoid this change

Comment (BrittanyIRL) [reply packages/templates/src/raw/travel.json:771]: noted for next time.

**Pull request merged**